.github/workflows/precommit.yaml

```diff
@@ -21,3 +21,7 @@ jobs:
         run: pip install -r requirements.txt
       - name: Run pre-commit
         run: pre-commit run --all-files
+      - name: Run pre-commit on the latest commit message
+        run:
+          pre-commit run gitlint --hook-stage commit-msg --commit-msg-filename
+          .git/COMMIT_EDITMSG
```

Comment:
Isn't this part of the previous line? Does it work this way?

Reply:
You're right, I forgot to set it as a multi-line. fixed.